site/content/docs/5.0/components/dropdowns.md

```diff
@@ -690,6 +692,89 @@ To align **left** the dropdown menu with the given breakpoint or larger, add `.d
 
 Note that you don't need to add a `data-bs-display="static"` attribute to dropdown buttons in navbars, since Popper isn't used in navbars.
 
+### Alignment options
+
+Taking most of the options shown above, here's a small kitchen sink demo of various dropdown alignment options in one place.
+
+{{< example >}}
+<div class="btn-group">
```

Comment:
Why are you not using just `dropdown` class here instead of `btn-group`. :thinking:

Reply:
Convenience here—we have some other styles for spacing our example button groups. I'll likely change these to dropdowns.

Reply:
Realizing also that we use them as they're `inline-block` and ensure proper alignment of the menu to the parent (e.g., a full-width `.dropdown` with a `.dropdown-menu-end` might not align properly).